### PR TITLE
initial sequence diagram for verified contact flow

### PR DESCRIPTION
**Pull request by azul**
references #29.

Using substeps with numbers like 3.a) 3.b)
for the separate parts of all the steps.
This way the sequence diagram can easily be compared
with the written instructions.

Diagram is fairly big and will probably result in a full page.
This should be fine as it allows comparing the instructions
and the diagram side by side.

### CODE REVIEW

File: images/contact.seq [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Note over Ua: 3.c) process AC header
+Ua -> Ub: 3.d) vc-auth-required message with AC header
+Note over Ub: 4.a) abort if key does not match FP from bootstrap
+Ub -> Ua: 4.b) vc-request-with-auth with Bob_FP and AUTH
+Note over Ua: 5.a) verify AUTH and key
+Note over Ua: 5.b) on failure alert user and abort
+Note over Ua: 6.a) signal success to user
+Ua -> Ub: 6.b) vc-contact-confirm message
+Note over Ua: 6.c) clear bootstrap data for INVITECODE
+Note over Ub: 7. signal success to user
```
Comment (hpk42) [inline images/contact.seq]: i suggest using capital "A" and "B" -- it's more visually distiguisahble than "Ub" and "Ua" .

**Comment by hpk42** [comment]
merged anyhow because the var naming is not a biggie. happy if you rename in another PR, survive if not :)

File: images/contact.seq (hunk 1 above)
Comment (azul) [reply images/contact.seq]: Applied it directly to master without an additional PR.

**Comment by ralphtheninja** [comment]
> merged anyhow because the var naming is not a biggie. happy if you rename in another PR, survive if not :)

This is c4 style a la the late @hintjens :)

**Pull request merged**